docs/integrations/index.md

```diff
@@ -0,0 +1,61 @@
+<!--
+title: Integrations
+menuText: Integrations
+description: Adding the AWS Observability Integration
+menuOrder: 3
+-->
+
+# Integrations
+Serverless Console uses an event driven approach to integrate with 
+third-party cloud services commonly used in Serverless Architectures. 
+
+This documentation gives you a reference for understanding how our Integration 
+process works and [what data is collected](./data-sources-and-roles.md) by Serverless Console. 
+
+## Adding your AWS Account
+Once you have [created an organization](../product/create-org.md) in Serverless Console
+you will be automatically be asked to add the AWS Account. Starting this process will start [syncing your resource](#synching-inventory) and will kick off the following in your AWS Account.
+
+1. Creation of [a set of IAM Roles](data-sources-and-roles.md#serverless-iam-roles) using CloudFormation. These roles are used to create additional infrastructure needed in your account. 
+
+1. Creation of an [EventBridge](./glossary.md#EventBridge) event bus and [CloudTrial](../glossary.md#cloudtrail) event trail for handling Lambda deploy events. This is used to keep your [resource inventory in synch](#synching-inventory) and [enable specific monitoring features](./enable-monitoring-features.md). 
+
+1. Creation of a [CloudWatch Metric Stream](../glossary.md#cloudwatch-metric-stream) for collection metrics across all your [Resources](../glossary.md#resources).
+
+1. Creation of two [Kinesis Firehose](../glossary.md#kinesis-firehose) for sending logs and metrics to Serverless Consoles.
+
+These steps may take a few minutes depending on the complexity of your account. 
+
+Note: this same process can be initiated from the CLI using [Serverless Framework](#onboarding-using-serverless-framework)
+
+### Synching Inventory
+Setting up the AWS Observability Integration automatically sets up a synch process to ensure that
```

Comment:
`AWS Integrations` and `AWS Observability Integrations`; I think I've seen both used. We should be consistent, and my vote is `AWS Integration`.

Reply:
Agreed - replaced on all files.